Prioritize/Java/Single Project/Gradle/prioritize.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+#### Prerequisite commands & installs
+# apt-get update
+# apt-get install -y curl git openjdk-8-jdk nano
+
+#### Clone your repo & run script
+# git clone <your repository> && cd ./<your repository>
+# chmod +x ./prioritize.sh
+# ./prioritize.sh
+
+#### Build application & check JAVA_HOME
+echo  $JAVA_HOME
```

Comment:
Consider changing this:
echo $JAVA_HOME
to this:
echo JAVA_HOME: $JAVA_HOME

This way it's easier to spot if JAVA_HOME wasn't set (because currently, if JAVA_HOME wasn't set, you'll just see a blank line in the log, nothing outstanding, you might miss it).